docs/docs.jsx

```diff
@@ -76,7 +76,22 @@ render(
         <Route path="compound-form(/*)" component={LayoutCompoundForm} />
         <Route path="ui-text(/*)" component={UITextView} />
       </Route>
-      <Route path="components">
+      <Route
+        path="components"
+        onChange={(prevState, nextState) => {
+          // If the component being rendered has changed, scroll to the top of the view container
+          if (prevState.location.pathname !== nextState.location.pathname) {
+            // HACK: Ideally we find a better way to target the main content view container,
+            // but this is the best approach I could find
+            const viewContainerList = document.getElementsByClassName(
+              prevState.components[0].cssClass.VIEW_CONTAINER,
+            );
+            if (viewContainerList && viewContainerList[0]) {
+              viewContainerList[0].scrollTo(0, 0);
```

Comment:
Is it possible to add scroll behavior here? Would love for this to be a `smooth` scroll if possible.

Reply:
should be able to do something like `{
  top: 0,
  left: 0,
  behavior: 'smooth'
}`

Reply:
Seems like an easy addition but I'm not sure if I find the UI natural. It feels like conflicting visual messaging between "scrolling to the specified content in this document" vs. "showing you the (new/distinct) specified content".
![2019-06-28 14 56 26](https://user-images.githubusercontent.com/8083680/60373419-f2e88b00-99b4-11e9-9aaa-f40a7359f875.gif)